Can't read index version 12; this IndexReader only supports index versions 2-3,6-11

The build currently fails with the next error:

```
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:3.10.0:build (default) on project quarkus-groovy-integration-tests-hibernate-orm-panache: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR] 	[error]: Build step io.quarkus.deployment.index.ApplicationArchiveBuildStep#build threw an exception: org.jboss.jandex.UnsupportedVersion: Can't read Jandex index from /META-INF/jandex.idx: Can't read index version 12; this IndexReader only supports index versions 2-3,6-11
[ERROR] 	at io.quarkus.deployment.index.IndexingUtil$MetaInfJandexReader.apply(IndexingUtil.java:288)
[ERROR] 	at io.quarkus.deployment.index.IndexingUtil$MetaInfJandexReader.apply(IndexingUtil.java:264)
[ERROR] 	at io.quarkus.paths.PathTreeVisit.process(PathTreeVisit.java:39)
[ERROR] 	at io.quarkus.paths.DirectoryPathTree.apply(DirectoryPathTree.java:106)
[ERROR] 	at io.quarkus.paths.ArchivePathTree$OpenArchivePathTree.apply(ArchivePathTree.java:262)
[ERROR] 	at io.quarkus.paths.PathTreeWithManifest.apply(PathTreeWithManifest.java:74)
[ERROR] 	at io.quarkus.paths.SharedArchivePathTree$CallerOpenPathTree.apply(SharedArchivePathTree.java:153)
[ERROR] 	at io.quarkus.deployment.index.IndexingUtil.indexTree(IndexingUtil.java:63)
[ERROR] 	at io.quarkus.deployment.index.ApplicationArchiveBuildStep.lambda$addMarkerFilePaths$1(ApplicationArchiveBuildStep.java:251)
[ERROR] 	at io.quarkus.bootstrap.classloading.PathTreeClassPathElement.apply(PathTreeClassPathElement.java:114)
[ERROR] 	at io.quarkus.deployment.index.ApplicationArchiveBuildStep.addMarkerFilePaths(ApplicationArchiveBuildStep.java:238)
[ERROR] 	at io.quarkus.deployment.index.ApplicationArchiveBuildStep.scanForOtherIndexes(ApplicationArchiveBuildStep.java:144)
[ERROR] 	at io.quarkus.deployment.index.ApplicationArchiveBuildStep.build(ApplicationArchiveBuildStep.java:105)
[ERROR] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
[ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
[ERROR] 	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:849)
[ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:256)
[ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:1583)
[ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:483) 
```

--- a/pom.xml
+++ b/pom.xml
@@ -32,6 +32,7 @@
     <quarkus.platform.version>${quarkus.version}</quarkus.platform.version>
     <groovy.version>4.0.21</groovy.version>
     <groovy-maven-plugin.version>3.0.2</groovy-maven-plugin.version>
+    <jandex-maven-plugin.version>3.1.7</jandex-maven-plugin.version>
     <hibernate-orm.version>6.4.0.Final</hibernate-orm.version>
     <docker-maven-plugin.version>0.44.0</docker-maven-plugin.version>
     <exec-maven-plugin.version>3.2.0</exec-maven-plugin.version>
@@ -124,6 +125,11 @@
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>exec-maven-plugin</artifactId>
           <version>${exec-maven-plugin.version}</version>
+        </plugin>
+        <plugin>
+          <groupId>io.smallrye</groupId>
+          <artifactId>jandex-maven-plugin</artifactId>
+          <version>${jandex-maven-plugin.version}</version>
         </plugin>
       </plugins>
     </pluginManagement>

